lib/app_config_reader.rb

```diff
@@ -0,0 +1,72 @@
+class AppConfigReader
+  attr_accessor :root_path
+
+  def initialize(
+    root_path: Rails.root,
+    s3_client: nil,
+    logger: Logger.new(STDOUT),
```

Comment:
I think this comma is a syntax error:

```
SyntaxError: /home/circleci/identity-idp/lib/app_config_reader.rb:8: syntax error, unexpected ')'
```
```suggestion
    logger: Logger.new(STDOUT)
```